gulpfile.js

```diff
@@ -55,7 +55,16 @@ gulp.task('min-qt', ['qt'], function() {
              .pipe(gulp.dest('./lib'));
 });
 
-gulp.task('build', ['qt', 'min-qt']);
+gulp.task('build', ['build-dev', 'build-rel']);
```

Comment:
`build` == `build-dev` and `build-rel`.
`build-rel` == `build-dev` and some actions.

This means that `build` == `build-rel`, so a separate `build-rel` entry isn't needed — just call it `build`.